<!-- -*- mode: XML -*- args="1" -->
<launch>

<!--设备编号1-->
<node pkg="lidar_driver" type="driver_node" name="driver_node" >
<param name="advertise_name" value="lidar_points" />
<param name="advertise_lidarScan_name" value="lidarScan" />

<param name="control_name" value="lidar_control"/>
<param name="device_ip" value="192.168.1.199" />
<param name="device_port" value="5000" />
<param name="pc_port"     value="6000" />
<param name="fps" value="10"/>
<param name="frame_id" value="/world"/>
<param name="section_agl_start_1" value="0"/>
<param name="section_agl_end_1" value="360"/>

<param name="section_agl_start_2" value="0"/>
<param name="section_agl_end_2" value="0"/>

<param name="section_agl_start_3" value="0"/>
<param name="section_agl_end_3" value="0"/>

<param name="section_agl_start_4" value="0"/>
<param name="section_agl_end_4" value="0"/>

<param name="section_agl_start_5" value="0"/>
<param name="section_agl_end_5" value="0"/>

<param name="point_cloud2_enable"   value="1"/>
<param name="laser_scan_enable"     value="1"/>

</node>

<!--设备编号2-->
<!--<node pkg="lidar_driver" type="driver_node" name="driver_node" >
<param name="advertise_name" value="lidar_points_1" />
<param name="control_name" value="lidar_control_1"/>
<param name="device_ip" value="192.168.1.200" />
<param name="device_port" value="5000" />
<param name="pc_port"     value="6000" />
<param name="fps" value="10"/>
<param name="frame_id" value="/world"/>
</node>-->

</launch>
